pages/register.js

```diff
@@ -0,0 +1,231 @@
+import React from 'react';
+import { Formik, Field, FieldArray, Form, ErrorMessage } from 'formik';
+import superagent from 'superagent';
+
+import { businessDays } from '../utils/businessDays';
+import { businessHours } from '../utils/businessHours';
+import { instruments } from '../utils/instruments';
+
+export default function RegisterPage() {
+  async function handleSubmit(values) {
+    console.log(values);
+    // const response = await superagent.post(
```

Comment:
no commented out code

Reply:
we will amend this and have use environment variables